doc/contributing.asciidoc

```diff
@@ -696,22 +696,15 @@ qutebrowser release
 
 * Make sure there are no unstaged changes and the tests are green.
 * Make sure all issues with the related milestone are closed.
-* Run `x=... y=...` to set the respective shell variables.
-
-* Update changelog (remove *(unreleased)*).
-* Adjust `__version_info__` in `qutebrowser/__init__.py`.
-* Commit.
-
-* Create annotated git tag (`git tag -s "v1.$x.$y" -m "Release v1.$x.$y"`).
-* `git push origin`; `git push origin v1.$x.$y`.
+* Update changelog (remove *(unreleased)*) and commit.
 * If committing on minor branch, cherry-pick release commit to master.
-* Create release on github.
-* Mark the milestone at https://github.com/qutebrowser/qutebrowser/milestones
-as closed.
-
-* Linux: Run `git checkout v1.$x.$y && ./.venv/bin/python3 scripts/dev/build_release.py --upload v1.$x.$y`.
-* Windows: Run `git checkout v1.X.Y; py -3 scripts\dev\build_release.py --asciidoc C:\Python27\python %userprofile%\bin\asciidoc-8.6.10\asciidoc.py --upload v1.X.Y` (replace X/Y by hand).
-* macOS: Run `git checkout v1.X.Y && python3 scripts/dev/build_release.py --upload v1.X.Y` (replace X/Y by hand).
+* Mark the milestone at https://github.com/qutebrowser/qutebrowser/milestones as closed.
+* Run `VERSION_TAG=$(./.venv/bin/python3 scripts/dev/update_version.py major | minor | patch)`.
+* Run `git push origin; git push $VERSION_TAG`.
+* Create new release via GitHub (required to upload release artifacts).
+* Linux: Run `git checkout $VERSION_TAG && ./.venv/bin/python3 scripts/dev/build_release.py --upload`.
+* Windows: Run `git checkout $VERSION_TAG; py -3 scripts\dev\build_release.py --asciidoc C:\Python27\python %userprofile%\bin\asciidoc-8.6.10\asciidoc.py --upload`.
```

Comment:
That's not going to work - `VERSION_TAG` was only set on Linux, so it can't be used directly on Windows/macOS as those are different machines/shells :wink:

What about making `update_version.py` print the appropriate `build_release.py` commands (and removing them from here) instead?

Reply:
> That's not going to work - VERSION_TAG was only set on Linux, so it can't be used directly on Windows/macOS as those are different machines/shells wink

I knew it looked too simple to be true, eh.

> What about making update_version.py print the appropriate build_release.py commands (and removing them from here) instead?

By that do you mean the entire string of the `build_release.py` command that needs to be run? I think this can be done.

Reply:
Yeah, printing the entire commands there instead of having them in `contribution.asciidoc` seems like a nice improvement - then I can copy-paste them from there with the right version numbers.

Reply:
I moved some of the commands to `update_version.py`, though obviously readability suffered, because they had to be PEP8-compliant. `contributing.asciidoc` was adjusted accordingly.